docs/content/users/usage/architecture.md

```diff
@@ -102,6 +102,27 @@ Files beginning with `.` are hidden because they shouldn’t be fiddled with; mo
 
 There’s only one global `.ddev` directory, which lives in your home directory: `~/.ddev` (`$HOME/.ddev`).
 
+!!!tip "Where is my global `.ddev` config?"
+    Use `ddev config global` (look at `global-config-path`) to check which location is currently used for the `.ddev` global directory.
+
+!!!tip "What if I don't want to clutter up my `$HOME` with a `.ddev` directory?"
```

Comment:
There's a risk here that people won't know that XDG_CONFIG_HOME is a system-wide setting that may affect other applications. Perhaps we should be saying things like "If XDG_CONFIG_HOME is set (with link) then DDEV's global directory will be stored there.

Reply:
Okay, I will add the link to the XDG documentation.